dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansLite-4.0/io.openliberty.enterpriseBeansLite-4.0.feature

```diff
@@ -12,7 +12,7 @@ Subsystem-Category: JakartaEE9Application
 -features=com.ibm.websphere.appserver.eeCompatible-9.0; ibm.tolerates:="10.0, 11.0", \
   com.ibm.websphere.appserver.contextService-1.0, \
   io.openliberty.ejbLiteCore-2.0, \
-  io.openliberty.jakarta.interceptor-2.0; ibm.tolerates:="2.1", \
+  io.openliberty.jakarta.interceptor-2.0; ibm.tolerates:="2.1 2.2", \
```

Comment:
Does this work without the comma delimiter?  I haven't seen it done this way before so I'm wondering if it's a typo.

Reply:
No.  That was bad on my part.  I have corrected it.